gst/tensor_mux/gsttensormux.c

```diff
@@ -400,6 +398,17 @@ gst_tensor_mux_collected (GstCollectPads * pads, GstTensorMux * tensor_mux)
   if (!tensor_mux->negotiated) {
     GstCaps *newcaps;
 
+    if (GST_IS_BUFFER (tensors_buf)) {
+      tensor_mux->tensors_config.info.num_tensors =
+          gst_buffer_n_memory (tensors_buf);
+    }
+
+    if (tensor_mux->tensors_config.info.num_tensors >= NNS_TENSOR_SIZE_LIMIT) {
```

Comment:
please check the condition (tensor_mux->tensors_config.info.num_tensors >= NNS_TENSOR_SIZE_LIMIT)
max size is 16 (tensor_mux->tensors_config.info.num_tensors == 16, failed)

Reply:
unreached, gst_buffer_n_memory() is never larger then 16.